### PR TITLE
[TTAHUB-879] fixes for consistent capitalization

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
App should use sentence case in most circumstances. Updated copy and tests where appropriate as flagged by UX.

## How to test
Confirm the following changes have been made:

**Fixes on Activity reports page:**

- "Activity reports – All regions" --> "Activity Reports – all regions"
- If "Activity reports – Region x", then "Activity Reports – Region x"

**Fixes on Regional TTA activity dashboard:**

- Title: Regional TTA activity dashboard
- "Total TTA Hours" --> "Total TTA hours"
- y axis: "Number of Hours" --> "Number of hours"
- x axis: "Date Range" --> "Date range"
- dropdown option: "High to Low" --> "High to low"
- Reasons in Activity Report column "# of Activities" --> "Number of activities"

**Fixes on Recipient records:**

- "Recipient Name" --> "Recipient name"
- "Program Specialist" --> "Program specialist"
- In the profile tab:
-- "Recipient Summary" --> "Recipient summary"
-- "Recipient Type" --> "Recipient type"
-- "Program Specialist" --> "Program specialist"
-- "Grant Specialist" --> "Grant specialist"
- In the TTA History tab
-- "Activity Reports" should be "Approved Activity Reports"

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-879

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/RecipientRecord/components/RecipientSummary.js [hunk 1]
```diff
@@ -65,9 +65,9 @@ export default function RecipientSummary({ summary, regionId }) {
             {summary.recipientId}
           </p>
         </div>
-        <RecipientInformationSection heading="Recipient Type" property="recipientType" grants={[{ recipientType: summary.recipientType }]} />
-        <RecipientInformationSection heading="Program Specialist" property="programSpecialistName" grants={summary.grants} distinct />
-        <RecipientInformationSection heading="Grant Specialist" property="grantSpecialistName" grants={summary.grants} distinct />
+        <RecipientInformationSection heading="Recipient type" property="recipientType" grants={[{ recipientType: summary.recipientType }]} />
+        <RecipientInformationSection heading="Program specialist" property="programSpecialistName" grants={summary.grants} distinct />
+        <RecipientInformationSection heading="Grant specialist" property="grantSpecialistName" grants={summary.grants} distinct />
```
Comment (GarrettEHill) [inline frontend/src/pages/RecipientRecord/components/RecipientSummary.js:70]: I think this change is wrong as these are proper names of titles and not sentence text.

File: frontend/src/pages/RecipientSearch/components/RecipientResults.js [hunk 2]
```diff
@@ -85,8 +85,8 @@ export default function RecipientResults(
         <thead>
           <tr>
             {renderColumnHeader('Region', 'regionId')}
-            {renderColumnHeader('Recipient Name', 'name')}
-            {renderColumnHeader('Program Specialist', 'programSpecialist')}
+            {renderColumnHeader('Recipient name', 'name')}
+            {renderColumnHeader('Program specialist', 'programSpecialist')}
```
Comment (GarrettEHill) [inline frontend/src/pages/RecipientSearch/components/RecipientResults.js:89]: I think this change is wrong as these are proper names of titles and not sentence text.

File: frontend/src/pages/RecipientRecord/components/RecipientSummary.js (hunk 1 above)
Comment (thewatermethod) [reply frontend/src/pages/RecipientRecord/components/RecipientSummary.js:70]: Confirmed with Adam Roux that these are supposed to be sentence text

File: frontend/src/pages/RecipientSearch/components/RecipientResults.js (hunk 2 above)
Comment (thewatermethod) [reply frontend/src/pages/RecipientSearch/components/RecipientResults.js:89]: Confirmed with Adam Roux that these are supposed to be sentence text

**Review by AdamAdHocTeam — COMMENTED** [review]
Do we want to make this one sentence case also?

image

**Review by AdamAdHocTeam — APPROVED** [review]
Looks good just need to merge main.

**Pull request merged**